torch/standalone/slim_tensor/utils.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <cstdint>
+#include <stdexcept>
+
+// OK to use c10 headers here because their corresponding cpp files will be
+// included in the final binary.
+#include <c10/core/Device.h>
+#include <c10/core/ScalarType.h>
+#include <c10/util/BFloat16.h>
+#include <c10/util/Half.h>
+
+#include <torch/standalone/slim_tensor/array_ref.h>
+
+namespace torch::standalone {
+inline size_t compute_numel(const ArrayRef& sizes) {
+  int64_t numel = 1;
+  for (auto& s : sizes) {
+    numel *= s;
+  }
+  return numel;
+}
```

Comment:
This is a duplicate of `c10::utils::multiply_integers`? 
https://github.com/pytorch/pytorch/blob/aec7cc60d7d290dfe143c7a67fdb5eea4f1ecb43/c10/util/accumulate.h#L45-L48

Reply:
Good point! I will revise.